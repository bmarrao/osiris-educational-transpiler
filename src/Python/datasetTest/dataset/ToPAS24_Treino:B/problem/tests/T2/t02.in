1 4
5 5
11 a 2 p 3 m 8 r
