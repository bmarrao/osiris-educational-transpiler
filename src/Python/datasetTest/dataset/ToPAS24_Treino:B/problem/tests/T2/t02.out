5 5 5 5 5
a a a a a a a a a a a
p p
m m m
r r r r r r r r
